src/indent.cpp

```diff
@@ -1312,11 +1338,26 @@ void indent_text(void)
       {
          prev = chunk_get_prev_ncnl(pc);
          if ((prev != NULL) &&
-             (prev->type == CT_BRACE_CLOSE) &&
+             ((prev->type == CT_BRACE_CLOSE) ||
+             (cpd.settings[UO_mod_case_brace].a & AV_REMOVE)) &&
```

Comment:
ok cool but I guess it would be better to be separate as it doesn't depend on the newly added options. right ?